content/applications/marketing/email_marketing/lost_leads_email.rst

```diff
@@ -0,0 +1,321 @@
+
+   .. image:: lost_leads_email/reasons.png
+      :align: center
+      :alt: A list of filter rules that exclude all lost reasons other than the desired reason.
+
+6. Finally, add a pair of :guilabel:`Active` filters to include both active and inactive leads for
+   the campaign.
+
+   .. important::
+      Adding both active **and** inactive lead records is necessary to capture the full scope of
+      lost leads in the database. Doing one without the other greatly impacts the number of
+      targetable records for the email campaign, and will not include a complete or accurate lost
+      leads audience.
+
+   Click the :guilabel:`(Add Branch)` icon to the right of the latest rule's row to add a pair of
+   :guilabel:`any of` rules. In the top rule's first field, select the :guilabel:`Active` parameter
+   from the drop-down menu. The rule will then automatically fill out to read: :guilabel:`Active`
+   *is* `set`.
+
+   For the first field of the bottom rule, once more select :guilabel:`Active` from the drop-down
+   menu and, this time, select :guilabel:`is not` from the operator drop-down menu, so the rule
+   reads: :guilabel:`Active` *is not* `set`.
+
+   .. image:: lost_leads_email/active.png
+      :align: center
+      :alt: A pair of Match Any Of filter rules that include both active and inactive leads.
+
+7. Now, with the domain section of the email campaign complete, create the body content of the email
+   using any of the premade stylized templates, or choose between the :guilabel:`Plain Text` or
+   :guilabel:`Start From Scratch` options for more granular control. For more information, refer to
+   the *Email Marketing* :ref:`documentation on how to create an email
+   <email_marketing/create_email>`.
+
+.. tip::
+   To save this set of filters for later use, click :guilabel:`Save as Favorite Filter`, enter a
+   name (such as *Lost Leads*), and click :guilabel:`Add`.
+
+   .. image:: lost_leads_email/favorite-filter.png
+      :align: center
+      :alt: The Save as Favorite Filter pop-up can save the lost leads criteria for later.
+
+8. Once all the components of the email campaign are complete:
+
+   - Click :guilabel:`Send` to immediately send the email;
+
+     OR
+   - Click :guilabel:`Schedule` to send the email at a future date and time.
+
+.. tip::
+   Consider using A/B Testing to send an alternate version of the email to a percentage of the
+   target leads. This can help determine what subject lines and body content produce the best
+   click-through rates before sending a final version to the remaining leads.
+
+   To do so, open the :guilabel:`A/B Tests` tab and check the box next to :guilabel:`Allow A/B
+   Testing`. Adjust the parameters as needed, and click :guilabel:`Create an Alternative Version`.
+
+   .. image:: lost_leads_email/ab-testing.png
+      :align: center
+      :alt: The A/B Tests tab with the Allow A/B Testing box checked to create an alternate version.
```

Comment:
Optional suggestion: maybe add some highlights to this screenshot.

Reply:
I see what you mean because there are many choices to make there. In this case, it makes sense for the user to experiment around on this tab, and I can write up a whole other doc on the different options and how-to's there.